### ISSUE TITLE
Add shh_* methods to RPC, eth_submitHashrate

### ISSUE THREAD

**Issue by zilm13** [ether-camp/ethereum-harmony#100]
`eth_submitHashrate` is not working, we need changes in EthereumJ to make it usable
`shh_*` methods need to be implemented
Everything else is up-to-date as of creating issue.
Not sure whether it's needed by somebody, so postpone with lowest priority.

**Comment by 3esmit** [comment]
Is this done?

**Comment by zilm13** [comment]
@3esmit no. Do you want to help with it?

**Comment by 3esmit** [comment]
Yes, I want to fork EthereumJ to connect with Status Network, but not sure if I would do it in top of Harmony.

**Comment by zilm13** [comment]
@3esmit If you pr it in ethereumj later, it will make most job done!
